libcontainer/specconv/spec_linux.go

```diff
@@ -411,8 +410,21 @@ func createLibcontainerMount(cwd string, m specs.Mount) (*configs.Mount, error)
 	return mnt, nil
 }
 
-// systemd property name check: latin letters only, at least 3 of them
-var isValidName = regexp.MustCompile(`^[a-zA-Z]{3,}$`).MatchString
+// isValidName checks if systemd property name is valid. It should consists
+// of latin letters only, and have least 3 of them.
+func isValidName(s string) bool {
```

Comment:
Related to my other comment (no need to fix in this PR); perhaps would be nice to have this return an error with the reason for it being invalid (too short or invalid character)

Reply:
Makes sense; opened #3285